config/config.go

```diff
@@ -17,8 +15,13 @@ func Get() (*Configuration, error) {
 	}
 
 	cfg = &Configuration{
-		BindAddr: ":8080",
+		BindAddr: "8080",
```

Comment:
In other services, we have the colon ":" as part of the BindAddr config, I would suggest keeping it, and removing it from `ListenAndServe`

Reply:
Thanks!
I think I was thrown by the idea of setting `export BIND_ADDR=:8081` (with the colon) instead of just `export BIND_ADDR=8081`. And I had some double colons hunting me 😆